Analysis/PIDTools/include/InputAlgorithm_dEdx.hh

```diff
@@ -0,0 +1,61 @@
+//#include <IMPL/ReconstructedParticleImpl.h>
+//#include <IMPL/TrackImpl.h>
+#include <InputAlgorithm.h>
+
+namespace cpid {
+
+  class InputAlgorithm_dEdx : InputAlgorithm {
```

Comment:
Is this a left over from development that can be removed? There is also a header with only a single `h`, so I am assuming this is not necessary(?)